### DISCUSSION TITLE
Error on retrieving data

### DISCUSSION THREAD

**Discussion by stevsharp** [simple-odata-client/Simple.OData.Client#880] [General]
Hello all..

I am getting this error when I am using the following core

 var returnList = await oDataClient.For<T>()
                .FindEntriesAsync();

Then this error is being returned ( For security reasons DTD is prohibited in this XML document. To enable DTD processing set the DtdProcessing property on XmlReaderSettings to Parse and pass the settings into XmlReader.Create method)

I am using blazor webAssembly core 6

Any ideas ?

Thanks

Spyros

**Comment by stevsharp** [comment]
Fixed , The problem was one the server

**Comment by dugsteen** [comment]
Hi. I'm having the same problem. Can you tell me what it is that fixed it on your server side? Thanks

**Comment by stevsharp** [comment]
Hi .. Please verify that the server Side ODATA is working correctly.
